### ISSUE TITLE
feat(accessibility): Update use of MenuList to List to resolve keyboard navigation issues

### ISSUE THREAD

**Issue by scarletmerlin123** [OpenCTI-Platform/opencti#17938]
## Use case

The use of MenuList causes issue with keyboard navigation when used in places where the List component is more appropriate. Menu components should be limited to be within popovers. 

## Current workaround

None

## Proposed solution

- Swap the usage of MenuList to List for components that are not contained within popovers 

## If the feature request is approved, would you be willing to submit a PR?

Yes
